metricsbp/baseplate_hooks.go

```diff
@@ -18,7 +21,12 @@ type CreateServerSpanHook struct {
 }
 
 // OnCreateServerSpan registers MetricSpanHooks on a server Span.
+//
+// It also configure the sentry scope with the current trace id.
 func (h CreateServerSpanHook) OnCreateServerSpan(span *tracing.Span) error {
+	sentry.ConfigureScope(func(scope *sentry.Scope) {
+		scope.SetTag("trace_id", strconv.FormatUint(span.TraceID(), 10))
```

Comment:
on second thought, this won't work as this is too global and will fail miserably in parallel request handling :(

Reply:
Done.